docs/extras/simp.md

```diff
@@ -0,0 +1,60 @@
+# Simp #
+
+The `simp` tactic works by applying "a conditional term rewriting system" (I don't really know what this means and would be interested to know more), to try and simplify the goal. You can actually watch what it's doing (which to the untrained eye looks like trying to apply lots of lemmas tagged with simp until it either makes progress or stops) by writing `set_option trace.simplify true` in your code. If you do this in the below
```

Comment:
short gloss for "conditional term rewriting system": this means that `simp` will rewrite subterms of the goal matching the LHS of one of its rules and replace with the RHS. "Conditional" means that rewrite rules are allowed to have preconditions, which `simp` attempts to discharge by simplifying to `true`.

Rather than `set_option trace.simplify true`, I would recommend mentioning `set_option trace.simplify.rewrite true` here since it's a bit easier to read. Maybe the other setting can be mentioned for completeness at the end.